MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -267,10 +267,39 @@ private void Setup()
             Dispose(false);
         }
 
+        void ClampMultisampleCountToSupportedByCurrentDevice()

```

Comment:
I would inline this call. IMO it's not worth a seperate function and GraphicsDevice is cluttered enough as it is.